dev/EnvironmentManager/ChangeTrackerHelper/ChangeTrackerHelper/ChangeTrackerHelper.h

```diff
@@ -0,0 +1,56 @@
+﻿#pragma once
+#include <windows.h>
+#include <appmodel.h>
+
+enum AreChangesTracked
+{
+    YES,
+    NO,
+    NOT_CHECKED
+};
+
+AreChangesTracked m_AreChangesTracked{NOT_CHECKED};
+
+inline bool DoesUnVirtualizedKeyExist()
+{
+    std::filesystem::path subKey = std::filesystem::path{ L"Software\\ChangeTracker" }
+
+}
+
+inline bool WillChangesBeTracked()
+{
```

Comment:
SUGGESTION to solve the global `g_areChangesTracked` issue

```
bool WillChangesBeTracked();
```

and in a source file

bool WillChangesBeTracked()
{
    static std::optional<bool> s_areChangesTracked{};

    if (!s_areChangesTracked.has_value())
    {
        if (scope == EnvironmentManager::Scope::Process)
        {
            s_areChangesTracked = false;
        }
        else
        {
            const auto packageFullName{ ::AppModel::Identity::GetCurrentPackageFullName() };
            if (GetCurrentPackageFullName.empty())
            {
                s_areChangesTracked = false;
            }
            else
            {
                m_packageFullName = std::wstring(packageFullName)
                s_areChangesTracked = true;
            }
        }

        return s_areChangesTracked.value();
    }
```

Reply:
This file is removed.